README.md

```diff
@@ -14,13 +15,24 @@ then in JAVA_OPTS add the
     use.test.url=true  
     ldap.server.url=ldap server e.g. ldap://ldap.itd.umich.edu:389/dc=umich,dc=edu  
     mcomm.group=group that can use this tool e.g. its-canvas-sections</code>
+5. Add the following 8 properties to sectionsToolFriend.properties: 
 
-5. Run this in local  
+	<code>
+	ctools.friend.url= friend URL either QA or PROD    
+	ctools.friend.contactemail=ctools contact email for QA or help desk for prod       
+	ctools.friend.referrer=Friend referrer url                            
+	ctools.friend.ksfilename=Key store file name	
+	ctools.friend.kspassword=Key store password		
```

Comment:
A password should be in a secure file.  In general it's better to keep the sensitive and non-sensitive properties in separate files.